static/uploads/plugins/faucet.svg

```diff
@@ -0,0 +1,34 @@
+<svg width="33" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please replace this with a smaller file
![faucet-min](https://user-images.githubusercontent.com/17264716/158365277-b01c5cfb-269c-4299-826d-d81221d44e86.svg)